packages/vite/src/node/plugins/importAnalysisBuild.ts

```diff
@@ -483,6 +485,16 @@ export function buildImportAnalysisPlugin(config: ResolvedConfig): Plugin {
             }
           }
 
+          s.append(`\
+function __viteFile(indexes) {
+  const fileDeps = ${JSON.stringify(
+    fileDeps.map((fileDep) =>
+      relativePreloadUrls ? path.relative(path.dirname(file), fileDep) : fileDep
+    )
+  )}
+  return indexes.map((index) => fileDeps[index])
+}`)
+
```

Comment:
While making some minor tweaks, I wonder if we should write this code minified since in the `generateBundle` phase the code is already minified, so this is appended as is.

Maybe it's not worth to do so since it's just a few bytes? Would love to get some opinions on this.

Reply:
I guess it depends on how much it makes the generated code harder to read? I do regularly (and others as well I guess) read generated code and I'm always happy if I can understand generated code :-).

Reply:
Aight I'll probably just rename the variables to squeeze some bytes but I'll leave the shape as is.